src/components/dashboard/DashboardPage.jsx

```diff
@@ -17,16 +19,21 @@ import CourseEnrollmentFailedAlert, { ENROLLMENT_SOURCE } from '../course/Course
 import { ProgramListingPage } from '../program-progress';
 import PathwayProgressListingPage from '../pathway-progress/PathwayProgressListingPage';
 import { features } from '../../config';
+import { getContentTypeSet, useContentHighlights } from '../search/content-highlights/data';
 
 export const LICENCE_ACTIVATION_MESSAGE = 'Your license was successfully activated.';
 
 const DashboardPage = () => {
   const { enterpriseConfig, authenticatedUser } = useContext(AppContext);
+  const { enterpriseConfig: { uuid: enterpriseUUID } } = useContext(AppContext);
   const { subscriptionPlan, showExpirationNotifications } = useContext(UserSubsidyContext);
   const { state } = useLocation();
   const history = useHistory();
   const [isActivationAlertOpen, , closeActivationAlert] = useToggle(!!state?.activationSuccess);
 
+  const { contentHighlights } = useContentHighlights(enterpriseUUID);
+  const contentTypeSet = getContentTypeSet(contentHighlights);
```

Comment:
nit: should `contentTypeSet` and/or `getContentTypeSet` be named a bit more explicit to "Highlights"? E.g., something like `highlightsContentTypeSet`. Reduces ambiguity for our future selves.